content/filters/html_filters.md

```diff
@@ -0,0 +1,94 @@
+# HTML Filters
+
+HTML filters wrap assets in HTML tags.
+
+1. TOC
+{:toc}
+
+## asset_url
+
+Add asset file to assets base url path.
+
+#### Input
+
+~~~ liquid
+  {{ 'images/large/' | asset_url }}
+~~~
+
+#### Output
+
+~~~ html
+  "/assets/themes/example_theme/images/large/"
+~~~
+
+## stylesheet_url
+
+Generate url with given stylesheet.
+
+#### Input
+
+~~~ liquid
+  {{ 'application.css' | stylesheet_url | stylesheet_tag }}
+~~~
+
+#### Output
+
+~~~ html
+  <link href="/assets/themes/1/stylesheets/application.css" rel="stylesheet" type="text/css" media="all">
+~~~
+
+## stylesheet_tag
+
+Generate a stylesheet tag based on given string and media type.
```

Comment:
on given stylesheet url
Actually is it even possible to pass 2 arguments to a filter (how to pass media?)